console/backend/src/main/java/org/frankframework/management/web/ShowConfigurationStatus.java

```diff
@@ -68,7 +70,7 @@ public Response getAdapters(@QueryParam("expanded") String expanded, @QueryParam
 	@Path("/adapters/{name}")
 	@Produces(MediaType.APPLICATION_JSON)
 	@Deprecated
-	public Response getAdapterOld(@PathParam("name") String name, @QueryParam("configuration") String configuration, @QueryParam("expanded") String expanded, @QueryParam("showPendingMsgCount") boolean showPendingMsgCount) {
+	public Response getAdaptersOld(@PathParam("name") String name, @QueryParam("configuration") String configuration, @QueryParam("expanded") String expanded, @QueryParam("showPendingMsgCount") boolean showPendingMsgCount) {
```

Comment:
```suggestion
	public Response getAdapterOld(@PathParam("name") String name, @QueryParam("configuration") String configuration, @QueryParam("expanded") String expanded, @QueryParam("showPendingMsgCount") boolean showPendingMsgCount) {
```
Yup, looks like it. Though this endpoint has been deprecated and doesn't show up in the list :)